File: blotztask-api/Models/ScheduledTasksDTO.cs
using System.Collections.Generic;

namespace BlotzTask.Models
{
    public class ScheduledTasksDTO
    {
        
        public List<TaskItemDTO> overdueTasks { get; set; }
        public List<TaskItemDTO> todayTasks { get; set; }
        public List<TaskItemDTO> tomorrowTasks { get; set; }
        public List<TaskItemDTO> weekTasks { get; set; }

        public Dictionary<int, List<TaskItemDTO>> monthTasks { get; set; }
    }


}

File: blotztask-ui/src/model/scheduled-tasks-dto.ts
import { TaskDetailDTO } from '../../../../model/task-detail-dto';

export interface ScheduledTasksDTO {
  overdueTasks: TaskDetailDTO[];
  todayTasks: TaskDetailDTO[];
  tomorrowTasks: TaskDetailDTO[];
  weekTasks: TaskDetailDTO[];
  monthTasks: Record<number, TaskDetailDTO[]>;
}
